src/pages/younghyunHan/Login/Login.js

```diff
@@ -1,7 +1,104 @@
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
 import './Login.scss';
 
 function Login() {
-  return <></>;
+  const [state, setState] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const [disabled, setDisabled] = useState(true);
+  const [style, setStyle] = useState({ opacity: 0.5, cursor: 'default' });
+
+  const navigate = useNavigate();
+
+  function handleIdInput(event) {
+    if (event.target.type === 'text') {
+      setState(event.target.value);
+    } else if (event.target.type === 'password') {
+      setPasswordValue(event.target.value);
+    }
+    handleButton();
+  }
+
+  const checkId = idValue => {
+    return idValue.includes('@') ? true : false;
+  };
+
+  const checkPw = pwValue => {
+    return pwValue.length >= 5 ? true : false;
+  };
```

Comment:
- 기능을 함수로 분리하는 부분은 좋음
- 삼항연산자로 사용할 필요 없이, 조건 자체로 boolean이 될 수 있음
```suggestion
  const checkId = idValue => {
    return idValue.includes('@');
  };

  const checkPw = pwValue => {
    return pwValue.length >= 5;
  };
```
- 추가적으로,
```js
const isInputValid = idValue.includes('@') && pwValue.length >= 5
```
이런 식으로 한꺼번에 변수로 만들어 처리할 수도 있음